omod/src/main/web/dashboardwidgets/dashboardwidgetscommons.service.js

```diff
@@ -83,7 +83,7 @@ export default class WidgetsCommons {
             today.setMonth(today.getMonth()-parseInt(maxAge));
         } else if( maxAge.indexOf('y') !== -1 ){
             maxAge = maxAge.replace('y', '');
-            today.setDate(today.getYear()-(parseInt(maxAge)));
```

Comment:
Was this something that was wrong that was discovered during testing? (Just curious)

Reply:
Nah, it came up when I was trying things out on a server. I configured a widget using max age `1y` and got unexpected results.